common.gypi

```diff
@@ -355,7 +355,7 @@
       }],
       [ 'OS in "linux freebsd openbsd solaris android aix cloudabi"', {
         'cflags': [ '-Wall', '-Wextra', '-Wno-unused-parameter', ],
-        'cflags_cc': [ '-fno-rtti', '-fno-exceptions', '-std=gnu++1y' ],
+        'cflags_cc': [ '-fno-rtti', '-fno-exceptions', '-std=gnu++1y', '-D__STDC_FORMAT_MACROS' ],
```

Comment:
Style: can you put these on separate lines so the sacred 80 column limit isn't violated?